src/app/(dashboard)/servers/[serverId]/server-members.tsx

```diff
@@ -8,7 +8,7 @@ import { ScrollArea } from "@/components/ui/scroll-area";
 export function ServerMembers({ id }: { id: Id<"servers"> }) {
   const members = useQuery(api.functions.server.members, { id });
   return (
-    <div className="flex flex-col max-w-80 w-full border-l p-4 bg-muted">
+    <div className="flex flex-col max-w-60 w-full border-l p-4 bg-muted">
```

Comment:
Setting max-w-full isn't useful if there's also a w-full tag. In this case, It would need a hard coded max-w in order to make a difference. This would have been a good example of utilizing a function graph tree to get the context of the parent component.